crates/bevy_color/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "bevy_color"
+version = "0.13.0"
+edition = "2021"
+description = "Types for representing and manipulating color values"
+homepage = "https://bevyengine.org"
+repository = "https://github.com/bevyengine/bevy"
+license = "MIT OR Apache-2.0"
+keywords = ["bevy", "color"]
+
+[dependencies]
+bevy_math = { path = "../bevy_math", version = "0.14.0-dev" }
+bevy_reflect = { path = "../bevy_reflect", version = "0.14.0-dev", features = [
+  "bevy",
+] }
+bevy_render = { path = "../bevy_render", version = "0.14.0-dev" }
+serde = "1.0.193"
```

Comment:
Specifying the version at the patch precision doesn't do anything functionally and actually makes it harder to keep everything in sync via find/replace. We currently specify serde "1.0" or "1" everywhere else (I prefer "1.0").

Reply:
Done.